extensions/libmodbus/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+include(${CMAKE_SOURCE_DIR}/extensions/ExtensionHeader.txt)
+
+include(ExternalProject)
+
+ExternalProject_Add(
+  libmodbus_external
+  PREFIX ${LIBMODBUS_ROOT}
+  GIT_REPOSITORY "https://github.com/stephane/libmodbus.git"
```

Comment:
Can you specify a tag or at least a commit here to make sure our builds doesn't get accidentally broken by changes in this repo?